R/FilterStates.R

```diff
@@ -133,7 +133,10 @@ FilterStates <- R6::R6Class( # nolint
         simplify = FALSE,
         function(state_list) {
           items <- state_list()
-          filtered_items <- Filter(f = function(x) x$is_any_filtered(), x = items)
+          filtered_items <- Filter(
+            f = function(x) x$is_any_filtered() && !identical(x$get_id(), sid),
```

Comment:
I wonder if we should allow sid to be a vector and have `!(x$get_id() %in% sid)` for future proofing?

Reply:
Let's keep `sid` a scalar now. If we get to thinking about filter-group then we can decide how to address this. I think almost everything needed is here already